nerfstudio/data/dataparsers/nerfstudio_dataparser.py

```diff
@@ -80,12 +80,9 @@ def _generate_dataparser_outputs(self, split="train"):
         num_skipped_image_filenames = 0
 
         for frame in meta["frames"]:
-            if "\\" in frame["file_path"]:
-                filepath = PureWindowsPath(frame["file_path"])
-            else:
-                filepath = Path(frame["file_path"])
+            filepath = PurePath(frame["file_path"])
```

Comment:
Thanks for the simplification

Reply:
This change actually doesn't work in the case where the data is processed on a Windows machine and trained / loaded on a Linux machine. The file paths in the dataset will be Windows file paths, but will be assumed to be a Linux file path by PurePath, because that's the OS of the machine loading the data. In this case, I think you still need to specify when the path loaded in is a Windows filepath with PureWindowsPath.

Reply:
Hmmm interesting. Can you elaborate how this fails?

I always thought that `frame["file_path"]` is a relative path, like `images/0000.jpg`, so that the full resolved path would be `self.config.data / frame["file_path"]`.

I admit I can't really test any Windows stuff since im on Mac, and I didn't really understand the failure for Windows in the first place.

Reply:
yeah, it was a with the process_data script because it would write out windows paths to the transforms.json file if run on Windows, which then couldn't be processed by a PurePosixPath. I think just changing process_data to always write posix paths should fix the issue as well and leave the code more clear.

Reply:
OK, I see your new PR. Thanks for the fix!